### DISCUSSION TITLE
Problemas con test 3-2 de la T0.

### DISCUSSION THREAD

**Discussion by Burbina200** [IIC2133-PUC/2024-1#45] [Tarea 0] [answered]
Hola! llevo un buen rato intentando hacer la parte 3.2 de la tarea (COORDINAR-PEDIDOS) y cuando testeaba con el test 3.2 me tiraba algunas diferencias con el output esperado. Hice un input mucho más pequeño para ir viendo como funcionaba el evento y por lo visto funcionaba correctamente. Solo por probar, intenté con los inputs de los siguientes test (3-2-inc y 3-hard) y con estos inputs mi código funcionó bien y devolvió el output esperado.

Podrían confirmarme si es que el input o el correct-output del test 3-2 están bien, ya que no me hace sentido la situación.

**Comment by benjavicente** [comment]
Hola, por favor, añade donde parte la diferencia del output y sube tu código a tu repositorio para poder ver la diferencia en el output.

**Reply by Burbina200** [reply, marked as answer]
Ya lo solucioné. Los inputs están bien.

Gracias de todas formas.
